lib/FrontendTool/FrontendTool.cpp

```diff
@@ -359,15 +359,15 @@ static void computeSwiftModuleTraceInfo(
     // FIXME: The behavior of fs::exists for relative paths is undocumented.
     // Use something else instead?
     if (fs::exists(moduleAdjacentInterfacePath)) {
-      err << "Found swiftinterface at\n" << moduleAdjacentInterfacePath
-          << "\nbut it was not recorded\n";
-      errorUnexpectedPath(err);
+      // This should be an error but it is not because of funkiness around
+      // compatible modules such as us having both armv7s.swiftinterface
+      // and armv7.swiftinterface in the dependency tracker.
+      continue;
```

Comment:
This one seems like you could still catch it, no? I wouldn't expect a case where you have a dependency on a swiftmodule and not a swiftinterface, even in the funky armv7s/armv7 case we talked about yesterday.

Reply:
The specific case that caught this error would have caught this error if the transitive import emits a .swiftinterface, and the second error if it doesn't emit a .swiftinterface. Just to be safe, for now, we're gonna disable both errors.

Reply:
Mmk, not going to argue against safety.